### ISSUE TITLE
Requesting Modality.Video does not work as I expected

### ISSUE THREAD

**Issue by tinsheep** [microsoftgraph/microsoft-graph-comms-samples#120]
I would like to "force" a video call to a user. I set the Modality to Video but it still gives me the option to pick up the call as both just audio AND audio+video. Is there any way to accomplish this currently?  Not sure if it's possible but perhaps a second bot answers that call on behalf of that user in audio+video modality? Thanks for any guidance.

**Comment by ksikorsk** [comment]
No, there is no way to force the user to include video when they answer.  You can request certain modalities, but the user can always choose which to accept.  I am not aware of anything in the teams pipeline to force user to answer with video.  @VinodRavichandran may know more.

**Comment by tinsheep** [comment]
OK Thanks Kamil - what about the idea of having a bot answer a call on a user's behalf and if I can do that can you answer with video?  Think of the user in this case as being a physical location or a kiosk.

**Comment by ksikorsk** [comment]
If it is a bot answering, you can force that it answers with video.  However, we don't have the capability of a bot answering on users' behalf today.

**Comment by tinsheep** [comment]
OK thank you.  Have you seen the new Consults feature/app in Teams?  In the external user join experience when they click on the link they get the video pre-selected once they give the browser permission to use the camera - they would have to specifically turn it off. I suppose that's different because the user is clicking on a meeting invite rather than a bot inviting them to join the meeting but that would be great. Seems like that modality does nothing in this scenario because I can still choose to answer the call with voice.

**Comment by ksikorsk** [comment]
I am unfamiliar with the new feature.  Even there it looks like you can turn it off.  It might even remember your last setting.  You are correct, joining a meeting is a different experience than receiving a call.
